src/seedu/addressbook/data/person/Tagging.java

```diff
@@ -0,0 +1,32 @@
+package seedu.addressbook.data;
+
+import java.util.ArrayList;
+
+public class Tagging {
+    
+    private static ArrayList<String> listOfTaggings;
+    
+    private boolean isAdd;
+    private String personName;
```

Comment:
assume the application allows to add multiple people with same name;
now, how will you associate the tag to the right person?